easybuild/easyconfigs/j/Jellyfish/Jellyfish-2.3.0-GCC-8.3.0-2.32.eb

```diff
@@ -0,0 +1,40 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia 
+# Homepage:     https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::     Robert Qiao <rob.qiao@flinders.edu.au> 
+# License::     GPLv3.0
+#
+# Notes::
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'Jellyfish'
+version = '2.3.0'
+
+homepage = 'http://www.genome.umd.edu/jellyfish.html'
+description = "Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA."
+
+toolchain = {'name': 'GCC', 'version': '8.3.0-2.32'}
```

Comment:
@robqiao the toolchain version should be just `8.3.0` (we removed the binutils version starting from this version).